modules/compliance-tests/src/smithy4s/compliancetests/CompatUtils.scala

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright 2021-2022 Disney Streaming
+ *
+ *  Licensed under the Tomorrow Open Source Technology License, Version 1.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     https://disneystreaming.github.io/TOST-1.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package smithy4s.compliancetests
+
+import cats.Monoid
+import cats.MonadError
+import cats.Monad
+
+private object CompatUtils {
+  type ME[F[_]] = MonadError[F, Throwable]
+}
+
+abstract class CompatUtils[F[_]: CompatUtils.ME] {
+  def raiseError[A](err: Throwable): F[A] =
+    MonadError[F, Throwable].raiseError(err)
```

Comment:
not sure why we need this (i.e. why not `MonadThrow[F].raiseError`, or `.raiseError[F, A]` - I think both exist in cats-core)

Reply:
I did not event know `MonadThrow` was part of cats, wow